lib/WasmReader/WasmSection.h

```diff
@@ -27,8 +27,8 @@ namespace Wasm
     {
         SectionFlag flag;
         SectionCode precedent;
-        char16* name;
-        char* id;
+        const wchar_t* name;
```

Comment:
Will change this back. The use of "mbstowcs_s" was giving pain in Linux, will look for a workaround.